conf/allofus.config

```diff
@@ -0,0 +1,60 @@
+includeConfig 'base.config'
+includeConfig 'container.config'
+
+// Define the number of CPUs based on memory requested
+def calculateCpus(memory) {
+    def mem
+    if (memory instanceof nextflow.util.MemoryUnit) {
+        mem = memory.toMega()
+    } else if (memory instanceof String) {
+        if (memory.toUpperCase().endsWith('G')) {
+            mem = memory[0..-2].toFloat() * 1024
+        } else if (memory.toUpperCase().endsWith('M')) {
+            mem = memory[0..-2].toFloat()
+        } else {
+            mem = memory.toFloat()
```

Comment:
I can raise an error instead of this additional logic. This was more so if the user happened to define memory incorrectly to traditional nextflow parameter requirements.